### PR TITLE
Add override for flask-principal

### PR DESCRIPTION
**Pull request by dhamaniasad**

### CODE REVIEW

**Comment by brettcannon** [comment]
flask-principal has not released support for Python 3 yet: that change to the tox.ini file was made in Nov 2014 and the last release was in Jun 2013

**Comment by dhamaniasad** [comment]
According to their changelog at https://github.com/mattupstate/flask-principal/blob/master/CHANGES#L11, they state that they added Python 3 support in version 0.4.0, which is the version that is currently up on PyPI.

File: caniusepython3/overrides.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "django-social-auth": "https://pypi.python.org/pypi/python-social-auth",
     "dnspython": "https://pypi.python.org/pypi/dnspython3",
     "extras": "https://pypi.python.org/pypi/extras",
+    "flask-principal": "https://github.com/mattupstate/flask-principal/blob/master/tox.ini#L2",
```
Comment (brettcannon) [inline caniusepython3/overrides.json]: Please update to `http://pythonhosted.org/Flask-Principal/changelog.html`

**Comment by brettcannon** [comment]
Please update the link and fix the merge conflict.

**Comment by dhamaniasad** [comment]
Done. I'll hold off on fixing the merge conflicts on the other pulls until this is merged in case this causes more conflicts.

**Comment by brettcannon** [comment]
Thanks!

**Pull request merged**